CRM/Aoservicelisting/Form/ProviderApplicationConfirm.php

```diff
@@ -186,19 +186,12 @@ public function submit($values) {
     // Delete all current addresses from the org contact and other contacts.
     self::removeAddressesLinkedToOrganization($organization['id']);
 
+    // Delete all current phones from the org contact and other contacts.
+    self::removePhonesLinkedToOrganization($organization['id']);
+
     //Ensure that the location type of all email addresses are work.
     self::setEmailsToWorkLocation($organization['id']);
 
-    $addId = civicrm_api3('Address', 'get', [
```

Comment:
This is defunct now that we are going with the tare down all addresses and replace